test/input/json/CustomInput.json

```diff
@@ -1,21 +1,78 @@
-{
-    "title_string": "Custom Input Example (using Pocono Test Example datafile)",
-    "geo_file_path": "../../test/input/geo/Channels/PoconoSampleData1/PoconoSampleRouteLink1.shp",
-    "key_col": 18,
-    "downstream_col": 23,
-    "length_col": 5,
-    "manningn_col": 20,
-    "manningncc_col": 21,
-    "slope_col": 10,
-    "bottomwidth_col": 2,
-    "waterbody_col": 8,
-    "waterbody_null_code": -9999,
-    "topwidth_col": 11,
-    "topwidthcc_col": 12,
-    "MusK_col": 7,
-    "MusX_col": 8,
-    "ChSlp_col": 12,
-    "terminal_code": 0,
-    "driver_string": "ESRI Shapefile",
-    "layer_string": 0
+{   
+    "run_parameters":
```

Comment:
@groutr 
The files checked in in this PR now include this demo file and everything needed to execute a full simulation: 
1. lateral inflows
2. reservoirs
3. initial conditions in the reservoirs
4. initial conditions in the channels

Reply:
(Pointing that out ... I'll be working soon on porting things over to v02)